src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -373,9 +399,9 @@ void GraphOptimizer::FuseFCAndWeightsDecompression(Graph &graph) {
         }
 
         // Fusion processing
-        fcNode->fuseDecompressionMultiply(multiplyConstNode);
+        fcNode->fuseDecompressionMultiplyPtr(multiplyConstNode);
```

Comment:
We still fuse decompression node itself, not ptr. So lets get rid of the suffix :)

Reply:
No problem, the suffix is removed)